app/controllers/base_controller.rb

```diff
@@ -0,0 +1,34 @@
+=begin
+Insights Service Approval APIs
+
+APIs to query approval service
+
+OpenAPI spec version: 1.0.0
+
+Generated by: https://github.com/swagger-api/swagger-codegen.git
+
+=end
+class BaseController < ApplicationController
+
+  def add_request
+    req = RequestCreateService.new(params[:workflow_id]).create(request_params)
+    json_response(req, :created)
```

Comment:
@hsong-rh 
The swagger-2.yaml is missing the request object definition coming back from addRequest

Reply:
@hsong-rh 
https://github.com/ManageIQ/approval-api/blob/408db9301c937bc2065b3e94da960721be05c39c/public/doc/swagger-2.yaml#L308

We need a 200 and response object back

Reply:
Response object is added with 201, the normal return code for successful `created` operation.